CDTDatastore/CDTDatastoreManager.m

```diff
@@ -26,6 +27,8 @@
 
 @interface CDTDatastoreManager ()
 
+@property NSMutableDictionary<NSString*, CDTDatastore*> *openDatastores;
```

Comment:
could this be made nonatomic given all reads/writes are inside a synchronized block?